zero/src/rpc/mod.rs

```diff
@@ -38,23 +39,44 @@ pub enum RpcType {
     Native,
 }
 
+/// The Jumpdest source type.
+#[derive(ValueEnum, Clone, Debug, Copy)]
+pub enum JumpdestSrc {
+    Simulation,
+    Zero,
+    Jerigon,
+}
```

Comment:
This is kind of confusing. Zero and Jerigon sound like they are the same source (or what else do you mean by Zero?) also we default jerigon nodes to Simulation so why do we have a Jerigon option?

Reply:
I agree. How about:

```rust
pub enum JumpdestSrc {
    ProverSimulation,
    ClientFetchedStructlogs,
    ServerFetchedStructlogs, // later
    Serverside,              // later
}
```

Reply:
@atanmarko What do you think about these names?

Reply:
Not sure if they are convenient to type into cli, but OK